pkg/kube/services/services.go

```diff
@@ -92,6 +96,49 @@ func GetServiceURLFromMap(services map[string]*v1.Service, name string) string {
 	return GetServiceURL(services[name])
 }
 
+func GetIstioVirtualService(namespace string, name string) (*unstructured.Unstructured, error) {
+	f := kubeclient.NewFactory()
+	cfg, err := f.CreateKubeConfig()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to get kubernetes config")
+	}
+	dynamicClient, err := dynamic.NewForConfig(cfg)
+	if err != nil {
+		return nil, errors.Wrap(err, "error building dynamic clientset")
+	}
+
+	//  Create a GVR which represents an Istio Virtual Service.
+	virtualServiceGVR := schema.GroupVersionResource{
+		Group:    "networking.istio.io",
+		Version:  "v1alpha3",
+		Resource: "virtualservices",
+	}
+	virtualService, err := dynamicClient.Resource(virtualServiceGVR).Namespace(namespace).Get(context.TODO(), name, meta_v1.GetOptions{})
+	return virtualService, err
+}
+
+//finding url from virtual services istio
```

Comment:
Change to // GetUrlFromVirtualService finds url from virtual services istio (gofmt will flag it otherwise, also that the general format for comments in go)